### ISSUE TITLE
Skeleton builder missing on GUI using docker install on Ubuntu 20.04.4

### ISSUE THREAD

**Issue by scoki211** [DeepLabCut/DeepLabCut#1993]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Hi there

Firstly, I tried installing DLC normally through pycharm, ran into a lot of dependency conflicts, finally managed to resolve it all and got the GUI up and running, latest version of DLC. Everything was going smoothly with labeling body parts, however once I finished this with my test set, I noticed there was no button for building skeletons. 

I couldn't figure this out, but had assumed it had something to do with my gtk library warnings. Tried installing the docker version (latest) and ran into the same issue. Contacted ITS at my university, who had play around with the docker version on their test rig, and then dug into the code and found that the skeleton builder had been commented out. 

I'm wondering why this is? Has that function been superseded? From watching the youtube tutorials it seemed as though that was a pretty integral part required for training. 

Any information would be very helpful

### Operating System

Ubuntu 20.04.4 LTS

### DeepLabCut version

Latest docker version

### DeepLabCut mode

multi animal

### Device type

GeForce RTX 3070

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by MMathisLab** [comment]
Hi @scoki211 check out #1984 which is the new GUI :D

**Comment by scoki211** [comment]
Thanks for that, I tried installing this version, but it seems to auto-install either a CPU version of tensor, or a version of tensor that is incompatible with my CUDA (10.1). I tried manually reinstalling the correct version of tensor for my CUDA but no dice, a million conflicts.

Guess I'll persist with just over connecting the labels in the skeleton section of the config.yaml and hope for the best.

**Comment by jeylau** [comment]
Hi @scoki211, the SkeletonBuilder used to be an integral part of training last year, when users were still asked to manually define the connections between keypoints. This is no longer required as DeepLabCut will default to learning all pairwise edges, and then select the best skeleton for inference when evaluating the network. So unless you want to force DeepLabCut to learn a particular skeleton (which I don't recommend, as it is very likely to perform worse than the one found in a data-driven manner), I would not be very concerned 😄.

**Comment by MMathisLab** [comment]
please now try running `pip install --upgrade --force-reinstall 'deeplabcut[gui]'==2.3rc1` in your env and/or docker. I will close, but please do post back!

**Comment by d45ha** [comment]
> Hi [@scoki211](https://github.com/scoki211), the SkeletonBuilder used to be an integral part of training last year, when users were still asked to manually define the connections between keypoints. This is no longer required as DeepLabCut will default to learning all pairwise edges, and then select the best skeleton for inference when evaluating the network. So unless you want to force DeepLabCut to learn a particular skeleton (which I don't recommend, as it is very likely to perform worse than the one found in a data-driven manner), I would not be very concerned 😄.

So does the config file still need to be updated with body marker connections?

**Comment by C-Achard** [comment]
@d45ha The skeleton is optional, and mostly serves for visualization, though there are some exceptions for models using part affinity field graphs, 

- In PyTorch the skeleton does not matter for models.
- In TensorFlow there is the following arg in `create_multianimaltraining_dataset`:
```python
"""
    paf_graph: list of lists, or "config" optional (default=None)
        Only for the TensorFlow engine.
        If not None, overwrite the default complete graph. This is useful for advanced users who
        already know a good graph, or simply want to use a specific one. Note that, in that case,
        the data-driven selection procedure upon model evaluation will be skipped.

        "config" will use the skeleton defined in the config file.
"""
```
where using "config" builds the the PAF graph from the config skeleton. The full pairwise graph is used by default.
